### DISCUSSION TITLE
未找到用户名输入框

### DISCUSSION THREAD

**Discussion by cikichen** [jxxghp/MoviePilot#2627] [问题咨询]
https://pt.btschool.club/

![QQ_1722846842554](https://github.com/user-attachments/assets/a20aabe6-f0de-4a3c-9436-56301b0a0bdc)

上面这个站点会更新会出现这个提示，无法自动更新

**Comment by cikichen** [comment]
![QQ_1722846957573](https://github.com/user-attachments/assets/17769244-7c5b-438a-9472-cd82dba9a495)

**Comment by wangjcheng001** [comment]
解决了吗，我添加麒麟站点也遇到通用的问题

**Comment by xw19930** [comment]
我也是这个问题，解决了嘛
